WalletWasabi.Gui/ViewModels/MainWindowViewModel.cs

```diff
@@ -69,6 +72,48 @@ public LockScreenViewModel LockScreen
 		public MainWindowViewModel()
 		{
 			Shell = IoC.Get<IShell>();
+
+			var global = Locator.Current.GetService<Global>();
+
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				var uiConfig = global.UiConfig;
+
+				Width = uiConfig.Width;
+				Height = uiConfig.Height;
+				WindowState = uiConfig.WindowState;
```

Comment:
Shouldn't we try if this works already for OSX and Linux with new Avalonia version?

Reply:
+

Reply:
OSX it isnt working, I will try and fix that in a separate PR